doc/theory/sh_basis.rst

```diff
@@ -14,9 +14,17 @@ Spherical harmonics are ortho-normal functions defined by:
 
     Y_l^m(\theta, \phi) = \sqrt{\frac{2l + 1}{4 \pi} \frac{(l - m)!}{(l + m)!}} P_l^m( cos \theta) e^{i m \phi}
 
-where $l$ is the band index, $m$ is the order, $P_l^m$ is an associated
-$l$-th degree, $m$-th order Legendre polynomial, and $(\theta, \phi)$ is the
-representation of the direction vector in the spherical coordinate.
+where $l$ is the order, $m$ is the degree, $P_l^m$ is an associated
+$l$-th order, $m$-th degree Legendre polynomial, and $(\theta, \phi)$ is the
+representation of the direction vector in the spherical coordinate. The relation
```

Comment:
`in the spherical coordinate.` : `in the spherical coordinate system.` or  `in spherical coordinates.`

Reply:
Addressed in 3f3f9ea83.